Application/EdFi.Ods.Features/OpenApiMetadata/Providers/OpenApiMetadataCacheProvider.cs

```diff
@@ -151,9 +152,8 @@ IEnumerable<OpenApiContent> CreateOpenApiMetadataUiSection()
                             x.Key,
                             new Lazy<string>(
                                 () =>
-                                    new OpenApiMetadataDocumentFactory(
-                                            new OpenApiMetadataDocumentContext(_resourceModelProvider.GetResourceModel()))
-                                        .Create(x.Value)),
+                                    new OpenApiMetadataDocumentFactory()
+                                        .Create(x.Value, new OpenApiMetadataDocumentContext(_resourceModelProvider.GetResourceModel()))),
```

Comment:
ultimately we will inject the document factory into the cache provider, and this will be _openApiMetadataDocumentFactory.Create(....)